packages/js/jest-browser-setup.js

```diff
@@ -0,0 +1,2 @@
+const fetchMock = require('jest-fetch-mock')
```

Comment:
I think it would better if this file was inside the test folder. Is there any reason to have it in the root folder?

Reply:
No reasons, was just easier to have it live next to `jest.config.js`

Reply:
Fixed!